### ISSUE TITLE
getting "Error while calling lua chunk" after clean install on MacOS

### ISSUE THREAD

**Issue by mohamed-ea** [siduck76/NvChad#2107]
**Describe the bug**
After a 1st install on MacOS 12.6.5 i'm getting this error each time i start nvim:
```
Error detected while processing /Users/mea/.config/nvim/init.lua:
E5113: Error while calling lua chunk: /Users/mea/.config/nvim/lua/core/init.lua:77: attempt to index field
 'fs' (a nil value)
stack traceback:
        /Users/mea/.config/nvim/lua/core/init.lua:77: in main chunk
        [C]: in function 'require'
        /Users/mea/.config/nvim/init.lua:1: in main chunk

```
**To Reproduce**
1. Go to Terminal
2. git clone https://github.com/NvChad/NvChad ~/.config/nvim --depth 1 && nvim

**Expected behavior**
Get into nvim without errors

**Screenshots**

**Desktop :**
 - MacOS 12.6.5
 - iTerm

**Additional context**
I tried to reset (delete .config folder) and install again but got the same error.

**Comment by siduck** [comment]
install neovim 0.9

**Comment by mohamed-ea** [comment]
thanks @siduck for the hint.

i installed nvim using brew, on the search on brew website they show 0.9.1 STABLE as the latest available version, but from terminal i only get 0.7 :/ 

When i tried `brew install --build-from-source  --HEAD neovim` i got an error : Could NOT find Lpeg (missing: LPEG_LIBRARY)

i believe it's related to nvim/brew but not NvChad.
